### ISSUE TITLE
AuthenticationSettings not available in MS ADAL plugin

### ISSUE THREAD

**Issue by senuraa** [ionic-team/ionic-native#2742]
**I'm submitting a ...**  (check one with "x")
[ ] bug report
[X] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://ionicworldwide.herokuapp.com/

**Current behavior:**
In order to use the broker mode, for Android need to access AuthenticationSettings (for android) which is available through Microsoft.ADAL.AuthenticationSettings. AuthenticationSettings is not available for ionic native plugin
**Expected behavior:**
Microsoft.ADAL.AuthenticationSettings should be available through the ionic native wrapper
**Steps to reproduce:**

**Related code:**

```
insert any relevant code here
```

**Other information:**

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):
Ionic:

   ionic (Ionic CLI)  : 4.1.1 (/usr/local/lib/node_modules/ionic)
   Ionic Framework    : ionic-angular 3.9.2
   @ionic/app-scripts : 3.1.11

Cordova:

   cordova (Cordova CLI) : 8.0.0
   Cordova Platforms     : android 7.0.0, browser 5.0.4, ios 4.5.5
   Cordova Plugins       : cordova-plugin-ionic-keyboard 2.1.2, cordova-plugin-ionic-webview 2.1.0, (and 11 other plugins)

System:

   Android SDK Tools : 26.1.1 (/Users/someuser/Library/Android/sdk/)
   ios-deploy        : 2.0.0
   NodeJS            : v10.9.0 (/usr/local/Cellar/node/10.9.0/bin/node)
   npm               : 6.4.0
   OS                : macOS High Sierra
   Xcode             : Xcode 10.0 Build version 10A255

**Comment by Digital-Chen** [comment]
Hi @senuraa, I am having the same headache. Have you found any workaround?

**Comment by senuraa** [comment]
@Digital-Chen yeah, you could use the cordova plugin without the ionic wrapper.
https://github.com/AzureAD/azure-activedirectory-library-for-cordova/tree/archive

**Comment by Digital-Chen** [comment]
Means I need to do it in the native part, after I export from Ionic? Sorry I am not very familiar with such scenario.

**Comment by senuraa** [comment]
@Digital-Chen No there's no need. You could just install it as a cordova plugin using the command below. 
`cordova plugin add cordova-plugin-ms-adal@0.8.x --save`
Then when you need to use the plugin add the code below before the class definition.
`declare let Microsoft: any;`

Finally, you could just use the plugin as below.
```
Microsoft.ADAL.AuthenticationSettings.setUseBroker(true).then( res => {
 //do anything here
})
```

[Check this link for more info](https://github.com/AzureAD/azure-activedirectory-library-for-cordova/tree/archive#how-to-use-the-library)

**Comment by Digital-Chen** [comment]
@senuraa So cool! I got it to work. Thank you so much.

**Comment by senuraa** [comment]
@Digital-Chen no worries! :) I don't think ionic will support ms-adal in the future, and Microsoft dropped support for cordova.. Better to find an alternative

**Comment by Digital-Chen** [comment]
That's very sad. I saw some discussions around this already. Let's see what comes out next.
